android/build.gradle

```diff
@@ -45,7 +45,7 @@ repositories {
     jcenter()
 }
 
-def FACEBOOK_SDK_VERSION = safeExtGet('facebookSdkVersion', '[7.0,8.0[')
+def FACEBOOK_SDK_VERSION = safeExtGet('facebookSdkVersion', '[7.0,9.0[')
```

Comment:
Should it be `'[7.0,9.0)'` ?

According to https://developers.facebook.com/docs/app-events/upgrade-guide?locale=en_US

Reply:
You are right - I'm not that great at the gradle version ranges, and I think we want all of 8.x but *exclusive* of 9 which is ')' thank you - I will update and re-push

https://docs.gradle.org/current/userguide/single_versions.html